### ISSUE TITLE
414 Request-URI Too Large

### ISSUE THREAD

**Issue by dsabsay** [idank/showthedocs#2]
Looks like there's a limit to how much text you can put in the query. Is there an easy way to fix this?

**Comment by idank** [comment]
Yeah, seems like there's an nginx config that can be tweaked. I'll give it a try when I'm from vacation. Thanks for the report!
